src/types/sequence-data.ts

```diff
@@ -9,3 +11,7 @@ export type SequenceData = {
   fragment_masses_y: number[]
   fragment_masses_z: number[]
 }
+
+export type SequenceDataDictionary = {
```

Comment:
Please correct me if I'm wrong: I don't see the need for this.. can we just have SequenceData with "coverage" and "maxCoverage" included? FLASHViewer can ignore these parameters.

Reply:
I implemented this because TaggerViewer has to handle multiple sequences. The key of the dictionary would be the index of the sequence. On your end you could probably just give a dictionary with a single key (e.g. 0) and sequence. 

This how I use it (`SequenceViewTagger.vue:189`):
```
sequence(): string[] {
      const key = this.selectedSequence;
      if (typeof key === 'number') {
        return this.streamlitDataStore.sequenceData?.[key]?.sequence ?? []
      }
      return []
    },
```

Reply:
Yes the keys have a significance for `FLASHTagger`as they refer to the id of the given protein, which is also used to map the sequence tags to the protein. You could definitely linearize it but I think this is the most intuitive solution?

Reply:
If the number has significance, then no array, but it can still be changed to `Record<Number, SequenceData>`. The type that you created is equivalent and is in line with the codebase so far. If you would like to keep the named type, you can always do something like `export type SequenceDataDictionary = Record<Number, SequenceData>` to create an alias.

Reply:
Thanks for letting me know that the two types are equivalent! I made the changes.